src/actions/searchActions.js

```diff
@@ -5,6 +5,7 @@ export const SEARCH_SET_SEARCH_RESULTS_COUNT = 'SEARCH_SET_SEARCH_RESULTS_COUNT'
 export const SEARCH_SET_SEARCH_RESULTS_PAGE = 'SEARCH_SET_SEARCH_RESULTS_PAGE';
 export const SEARCH_SET_SEARCH_RESULTS = 'SEARCH_SET_SEARCH_RESULTS';
 export const SEARCH_SET_CROSS_REFERENCE_RESULTS = 'SEARCH_SET_CROSS_REFERENCE_RESULTS';
+export const SERACH_SET_CURIE_MAIN_PDF_IDS_MAP_RESULTS = 'SERACH_SET_CURIE_MAIN_PDF_IDS_MAP_RESULTS';
```

Comment:
Not important but typo in SEARCH. At least it is used the same throughout though so no harm. (yeah I know the typo king commenting on this!! the cheek)

Reply:
Fixed typo here https://github.com/alliance-genome/agr_literature_ui/pull/330